lib/exirc/utils.ex

```diff
@@ -78,7 +78,17 @@ defmodule ExIrc.Utils do
       |> Enum.filter(fn(arg) -> arg != [] end)
       |> Enum.map(&trim_crlf/1)
       |> Enum.map(&List.to_string/1)
-    %{msg | args: args}
+
+    # Handle Slack's inappropriate RPL_TOPIC when they should send RPL_NOTOPIC:
+    # https://github.com/bitwalker/exirc/issues/51
+    case {msg, args} do
+      { %IrcMessage{cmd: "332"}, [_] } ->
+        # We shouldn't end up in the situation where we have a "332" but only one arg.  Turn this into a 331.
+        [channel, nick] = hd(msg.args)
```

Comment:
This can be moved into the pattern for the case clause, but I can take care of that after merging if time is tight.